### PR TITLE
Adding my project named Animations

### PR DESCRIPTION
**Pull request by UsmanAhmadSaeed**
This is a simple animations project to compile different animations that can be used on websites/apps. I believe this will be a great starter for new contributors as well as playground for skilled ones.

Let me know if there's anything else needed.

P.S. Project domain expired, working on re-registeration, won't take long. Will be looking to add more maintainers as the project grows.
Also, if it's possible, please add the hacktoberfest-accepted label as well. (not the best OS contribution, but trying to catch up this year, after finishing my exams)

### CODE REVIEW

File: _data/projects/Animations.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Animations
+desc: An open source collection of animations, from loaders, to menus, and scroll transitions, and anything in between
+site: https://animations.gq
```
Comment (ritwik12) [inline _data/projects/Animations.yml:3]: @UsmanAhmadSaeed Please change the link to the site or let us know when it is up. putting on hold to merge for the working site link

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Animations.yml (hunk 1 above)
Comment (UsmanAhmadSaeed) [reply _data/projects/Animations.yml:3]: It's up. Thanks!
Currently not a site, just presenting the markdown file, but issue is open for a simple site development.

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/Animations.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/Animations.yml:3]: @UsmanAhmadSaeed https://animations.gq/ is not working. not presenting a markdown file

Comment (ritwik12) [inline _data/projects/Animations.yml:3]: @UsmanAhmadSaeed https://animations.gq/ 
Not working, doesn't seem to be a markdown file too

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@UsmanAhmadSaeed Thanks!!

**Pull request merged**